redfish-core/lib/log_services.hpp

```diff
@@ -1579,9 +1587,22 @@ inline void requestRoutesDBusEventLogEntryCollection(App& app)
                                         }
                                         resolved = *resolveptr;
                                     }
+                                    else if (propertyMap.first ==
+                                             "ServiceProviderNotify")
+                                    {
+                                        bool* servicepnptr = std::get_if<bool>(
```

Comment:
Doesn't follow our naming here: serviceProviderNotifiedptr

Reply:
Let's get https://github.com/ibm-openbmc/bmcweb/pull/114 in and we can rebase on top

Reply:
fixed now by resolving merge conflict